Formula/grep.rb

```diff
@@ -5,7 +5,7 @@ class Grep < Formula
   mirror "https://ftpmirror.gnu.org/grep/grep-3.8.tar.xz"
   sha256 "498d7cc1b4fb081904d87343febb73475cf771e424fb7e6141aff66013abc382"
   license "GPL-3.0-or-later"
-  revision 1
+  revision 2
```

Comment:
```suggestion
  revision 1
```
Let's skip the `revision` bump; this change isn't worth forcing everyone to reinstall the formula.